docs/UserDefinedLanguageServer.md

```diff
@@ -87,14 +87,46 @@ The `Debug tab` is available when you have created the language server definitio
 level Trace used in LSP console.
 
 ### Using template
+Template can be used to quickly create user defined language server pre-filled 
+with server name, command, mappings and potential configurations.
 
-The `Template combo-box` provides some `language servers templates`, pre-filled with server name, command, mappings and potential configuration. 
+#### Default template
+The `Template combo-box` provides some default language servers templates (located in templates directory classpath), 
+pre-filled with server name, command, mappings and potential configuration.
 
-![New Language Server with Template](./images/user-defined-ls/NewLanguageServerWithTemplate.png)
+If the template folder contains a `README.md` file, you can press the help button to open the instructions.
 
- * [Clojure LSP](./user-defined-ls/clojure-lsp.md)
- * [CSS Language Server](./user-defined-ls/vscode-css-language-server.md)
- * [Go Language Server](./user-defined-ls/gopls.md) 
- * [HTML Language Server](./user-defined-ls/vscode-html-language-server.md)
- * [Scala Language Server (Metals)](./user-defined-ls/metals.md)
- * [TypeScript Language Server](./user-defined-ls/typescript-language-server.md)
+![New Language Server with Default Template](./images/user-defined-ls/NewLanguageServerWithDefaultTemplate.png)

```

Comment:
* Move the screenshot before L97 (If the template folder contains a `README.md` file, you can press the help button to open the instructions.)
 * Update the screenshot with the new UI and draw a red square to highlight list of default templates